src/components/CalendarDay.jsx

```diff
@@ -41,61 +39,61 @@ export default class CalendarDay extends React.Component {
     return shallowCompare(this, nextProps, nextState);
   }
 
-  handleDayClick(day, modifiers, e) {
-    this.props.onDayClick(day, modifiers, e);
+  onDayClick(day, e) {
+    this.props.onDayClick(day, e);
```

Comment:
Not a big deal, but it'd be good to destructure out the prop and then call it, so that its `this` value won't be `this.props` (same throughout)